infinite_tracing/lib/infinite_tracing/proto/infinite_tracing_services_pb.rb

```diff
@@ -1,12 +1,10 @@
-# encoding: utf-8
 # This file is distributed under New Relic's license terms.
 # See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
 # frozen_string_literal: true
 
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
 # Source: infinite_tracing.proto for package 'com.newrelic.trace.v1'
 # Original file comments:
-# encoding: utf-8
```

Comment:
LOL that does seem quite dramatic.
Changing the comments is totally fine though, the part it doesn't want edited is the ruby code because it was autogenerated from the .proto files. We added comments below here in the file when we first made infinite tracing to explain what things were